README.md

```diff
@@ -102,7 +102,7 @@ Continue with the installation.
 <img height="400" src="templates/img/conda_powershell.png" width="500"/>
 </p>
 
-* If macOS user, open terminal. 
+* If macOS user, open terminal.
```

Comment:
Do the commands below still hold, or is there now an easier process?

Reply:
It's the same process as before!